libs/pika/init_runtime/include/pika/init_runtime/init_runtime.hpp

```diff
@@ -156,7 +156,7 @@ namespace pika {
     ///
     /// Until @ref pika::finalize has been called, @ref pika::stop will not return. This function
     /// exists to distinguish between the runtime being idle but still expecting work to be
-    /// scheduled on it and the runtime being idle and ready to be shutdown. Unlike @pika::stop,
+    /// scheduled on it and the runtime being idle and ready to be shutdown. Unlike @ref pika::stop,
```

Comment:
Thanks for spotting this! Do you mind removing this change here, since it looks like this isn't working the way it should (my fault). In a few cases we've used `@p` to make it monospace, but no link. In most other places we've used `@ref` but it doesn't make it monospace and it doesn't seem to link it correctly as well. I can look at this outside of this PR.

Reply:
https://github.com/pika-org/pika/pull/1298 fixes the above separately.